### PR TITLE
[codex] add worktrunk bulk management

### PR DESCRIPTION
**Pull request by sam-phinizy**
## Summary
- Add archived-session-style filtering and dense selectable rows to the Worktrunk worktree list.
- Add quick-select actions for merged and prunable worktrees.
- Add bulk remove actions for worktrees and worktree + branch, while preserving main-worktree and active-session safety guards.

## Validation
- `npm run test -- src/lib/components/__tests__/WorktrunkPanel.test.ts`
- `npm run check`

Note: the targeted Vitest run still emits existing jsdom canvas/localstorage warnings, but all tests pass.

## Summary by CodeRabbit

* **New Features**
  * Added searchable filter for worktrees by branch or path, with clear control.
  * Introduced multi-select with per-row checkboxes, select-all (indeterminate support), and a bulk action toolbar.
  * Header quick-select options for "Select merged" and "Select prunable".
  * Bulk removal with optional branch deletion, confirmation, and partial-failure reporting.
  * Worktree path display can be toggled off.

* **Tests**
  * Expanded test coverage for filtering, selection behaviors, bulk removal flows, and edge cases.

### CODE REVIEW

File: src/lib/components/WorktrunkPanel.svelte [hunk 1]
```diff
@@ -594,79 +878,208 @@
...
+                checked={allVisibleSelected}
+                disabled={visibleRemovableWorktrees.length === 0 || bulkPending}
+                onchange={toggleAllVisible}
+                data-testid="worktrunk-select-all"
+              />
+              <span>
+                {#if filterText}
+                  Select {filteredWorktrees.length} match{filteredWorktrees.length === 1 ? "" : "es"}
+                {:else}
+                  Select all
```
[the remark this answers is not in the document]
Comment (sam-phinizy) [reply src/lib/components/WorktrunkPanel.svelte]: Fixed in 8ddf226. The select-all label now reflects the removable visible worktrees it actually operates on, using removable-count wording for both filtered and unfiltered states.

File: src/lib/components/WorktrunkPanel.svelte [hunk 2]
```diff
@@ -352,6 +444,102 @@
...
+          await removeWorktree(repo, wt.path, alsoBranch);
+          succeeded.push(wt.path);
+        } catch (err) {
+          failures.push({
+            branch: wt.branch,
+            error: typeof err === "string" ? err : String(err),
+          });
+        }
+      }
+      if (succeeded.length > 0) {
```
[the remark this answers is not in the document]
Comment (sam-phinizy) [reply src/lib/components/WorktrunkPanel.svelte]: Fixed in 8ddf226. Bulk removal now skips the stale repo refresh unless the repo is still current, and the test covers switching active repos during the operation.

File: src/lib/components/WorktrunkPanel.svelte [hunk 3]
```diff
@@ -594,79 +878,208 @@
...
+                onchange={toggleAllVisible}
+                data-testid="worktrunk-select-all"
+              />
+              <span>
+                {#if filterText}
+                  Select {filteredWorktrees.length} match{filteredWorktrees.length === 1 ? "" : "es"}
+                {:else}
+                  Select all
+                {/if}
+              </span>
```
[the remark this answers is not in the document]
Comment (sam-phinizy) [reply src/lib/components/WorktrunkPanel.svelte:993]: Already addressed in 8ddf226. The select-all copy now uses visibleRemovableWorktrees.length so the displayed count matches what the checkbox can select.
